updates/testing.json

```diff
@@ -21,18 +21,18 @@
       }
     },
     {
-      "version": "31.20200118.2.0",
+      "version": "31.20200127.2.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "31.20200127.2.0",
+      "version": "31.20200210.2.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1580738400,
+          "start_epoch": 1578837600,
```

Comment:
This looks off? CI prints:

```
testing rollout of 31.20200210.2.0 scheduled for 2020-01-12 14:00:00 UTC for 1440m (24.0h)
```

Reply:
nope that was a mistake.. 01/12 vs 02/12 fixed now